panic: runtime error: invalid memory address or nil pointer dereference with aztfexport v0.13.0

Hi,

OS: macOS Ventura v13.4.1
aztfexport: v0.13.0

When I try to use the latest `aztfexport v0.13.0` to export a resource group created by [startstopv2-deployments](https://github.com/microsoft/startstopv2-deployments) template.  I got the error:

```shell
aztfexport resource-group <myResourceGroup>

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x0 pc=0x103504444]

goroutine 601 [running]:
github.com/magodo/aztft/aztft.queryType({0x14001b6e780?, 0x102ec4f4c?}, 0x14001791dd8)
	github.com/magodo/aztft@v0.3.1-0.20230713032200-ac231869c31f/aztft/aztft.go:138 +0x1a4
github.com/magodo/aztft/aztft.QueryTypeAndId({0x14001b6e780?, 0x2020202020202020?}, 0x3a226b636f6c6222?)
	github.com/magodo/aztft@v0.3.1-0.20230713032200-ac231869c31f/aztft/aztft.go:48 +0x2c
github.com/Azure/aztfexport/internal/resourceset.AzureResourceSet.ToTFResources.func2()
	github.com/Azure/aztfexport/internal/resourceset/azure_resource_set.go:78 +0x9c
github.com/magodo/workerpool.(*workPool).Run.func1(0x2020202020202020?)
	github.com/magodo/workerpool@v0.0.0-20230119025400-40192d2716ea/workerpool.go:75 +0x78
created by github.com/magodo/workerpool.(*workPool).Run
	github.com/magodo/workerpool@v0.0.0-20230119025400-40192d2716ea/workerpool.go:66 +0x50
```

I don't have any issue when I used `aztfexport v0.12.0`.  I don't think it's the same problem as [issue#257](https://github.com/Azure/aztfexport/issues/257)

Is this a new bug?

Cheers,

Tao

--- a/go.mod
+++ b/go.mod
@@ -21,7 +21,7 @@ require (
 	github.com/hexops/gotextdiff v1.0.3
 	github.com/magodo/armid v0.0.0-20220923023118-aec41eaf7370
 	github.com/magodo/azlist v0.0.0-20230518102903-58631213ca2c
-	github.com/magodo/aztft v0.3.1-0.20230713032200-ac231869c31f
+	github.com/magodo/aztft v0.3.1-0.20230718024242-59215c807c21
 	github.com/magodo/spinner v0.0.0-20220720073946-50f31b2dc5a6
 	github.com/magodo/terraform-client-go v0.0.0-20230323074119-02ceb732dd25
 	github.com/magodo/textinput v0.0.0-20210913072708-7d24f2b4b0c0

